templates/repo/issue/new_form.tmpl

```diff
@@ -73,14 +73,10 @@
 			<div class="ui labels list">
```

Comment:
Could all of this code go in shared template if it is the same in both places? Everything inside 
```<div class="ui labels list">```

Reply:
You're right, I missed the fact that this parent matches in both.

Reply:
Done